### ISSUE TITLE
Can we address constituencies?

### ISSUE THREAD

**Issue by stenzer** [DemocracyEarth/paper#269]
I'm thinking of a "constituency" as the group of people that have standing in an election. There are natural constituencies, like the residents of a county in a county election. There are ad-hoc constituencies, which may grow or change over time too. And there are constituencies that might be more accurately described as "interested parties."  How do votes and constituencies relate, and how are constituencies addressed in the proposal?

**Comment by metamerman** [comment]
I don't see that it is, which IMHO is a good thing: Including support for "constituencies", which I generally translate  as support for tribalism, is exactly the *opposite* of what we should be doing. Your proposal boils down to building in support for oligarchy which is fundamentally incompatible with democracy (if you haven't, I'd again recommend reading the help system for https://www.proxyfor.me to get some clue as to why this is).

**Comment by Tasty213** [comment]
@metamerman I disagress some nissues have to be fought on a local level. For instanc I may wish to delegate my vote on how best to deliver aid to people hit by natural disasters to an expert in that field, but if i may want to delegat my vote on wether the state should pay to dredge the local ditches to prevent flooding. There should at loeast be a '#local' or '#wherever_you_happen_to_live'. So not nessecerily preset constituincies but local groups banding together would be good.

**Comment by metamerman** [comment]
Right: Locality != "constituency" except in the rare (non-existent?) case where the locality is completely homogeneous. Certainly some laws/policies and budgeting will only apply at the local level, but that's a completely unrelated issue to that of intentionally dividing even the *local* community up into different "tribes" who are expected to compete with each other. I note that the original poster never bothered to respond to the criticism that their proposal is designed to facilitate tribalism which is the single most broken thing in our existing governing systems because it effectively *prevents* cooperation from arising spontaneously (i.e., those who "don't have a dog in the fight" are effectively shut out of the decisionmaking process where they would naturally serve as a coordinating and moderating influence). Democracy only works when *everybody* participates in *every* decision, even if it doesn't affect them directly, and AFAIK https://www.proxyfor.me/ is the only decisionmaking system that supports this without imposing totally impractical demands on The People. Certainly no "Liquid Democracy" derivative can do this because the design of that delegation system is fatally flawed...
